packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -369,17 +370,6 @@ if you do not wish to download ${input('apk')}.
     return String(value);
   }
 
-  async getDefaultArtifactType(platform: string, useOriginalDefault?: boolean): Promise<Array<string>> {
```

Comment:
This was deleted to instead download all available artifacts that the build has available to it.